pkg/csi_driver/gcfs_driver.go

```diff
@@ -267,3 +308,130 @@ func (driver *GCFSDriver) Run(endpoint string) {
 	}
 	s.Wait()
 }
+
+func runMultishareReconciler(driverConfig *GCFSDriverConfig) {
```

Comment:
Can we break this into 2 functions, one which initializes the reconciler object and sets up the driverConfig
and one for the run which starts the leader-election watch and the run() function of the reconciler?